sections/git_commit_time.zsh

```diff
@@ -0,0 +1,48 @@
+#
+# Git commit_time
+#
+# Show time since previous commit
+# Inspired from "hyperzsh" theme
+# ------------------------------------------------------------------------------
+# Configuration
+# ------------------------------------------------------------------------------
+
+SPACESHIP_GIT_COMMIT_TIME_SHOW="${SPACESHIP_GIT_COMMIT_TIME_SHOW=true}"
+SPACESHIP_GIT_COMMIT_TIME_PREFIX="${SPACESHIP_GIT_COMMIT_TIME_PREFIX="since "}"
+SPACESHIP_GIT_COMMIT_TIME_SUFFIX="${SPACESHIP_GIT_COMMITT_TIME_SUFFIX=}"
```

Comment:
This typo is not fixed as [requested](https://github.com/denysdovhan/spaceship-prompt/pull/316#discussion_r162785969) by @maximbaz